parser/csv_tree_parser_test.go

```diff
@@ -175,6 +175,18 @@ func TestParseNodes(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "when quotation, then works",
+			in:   "a\"b\",1,1",
```

Comment:
Quotes are possible in paths now, so I changed the tests accordingly.

Not sure if this has any side effects, but can't see any at first glance.

If we don't want double quotes to be allowed, the `LazyQuotes` should be turned off again.